encode_test.go

```diff
@@ -656,6 +656,18 @@ func (s *S) TestSetIndent(c *C) {
 	c.Assert(buf.String(), Equals, "a:\n        b:\n                c: d\n")
 }
 
+// This test is providing coverage for SetIndentSequences function.
+// When we are setting that to false the sequences are not indented.
+func (s *S) TestSetIndentSequences(c *C) {
+	var buf bytes.Buffer
+	enc := yaml.NewEncoder(&buf)
+	enc.SetIndentSequences(false)
```

Comment:
should there be a test for setting it to true?

Reply:
yes, it's actually true (well the inverse under the hood) by default so the above tests for sequences cover it.  We can add an explicit set to true and make sure it works.  It's not a bad idea.